dataline-config/init/src/main/resources/config/DESTINATION_CONNECTION_SPECIFICATION/e28a1a10-214a-4051-8cf4-79b6f88719cd.json

```diff
@@ -5,10 +5,31 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "title": "BigQuery Destination Spec",
     "type": "object",
-    "required": ["serviceAccount"],
+    "required": [
+      "project_id",
+      "dataset_id",
+      "credentials_json",
+      "disable_collection",
+      "replication_method"
+    ],
     "additionalProperties": false,
     "properties": {
-      "serviceAccount": {
+      "project_id": {
```

Comment:
Could you add descriptions to these params? I'm not sure what `validate_records` or `disable_collection` means